x86_64/src/regs.rs

```diff
@@ -311,6 +323,9 @@ mod tests {
 
         assert_eq!(0xa003, read_u64(&gm, layout::PML4_START));
         assert_eq!(0x83, read_u64(&gm, layout::PDPTE_START));
```

Comment:
This is the offending line that wasn't properly pulled from the crosvm fix.

```suggestion
        assert_eq!(0xb003, read_u64(&gm, layout::PDPTE_START));
```

Reply:
@acatangiu  my bad.. first time doing anything with rust